src/main/java/frc/robot/commands/OnBridgeUntilMovement.java

```diff
@@ -21,6 +20,14 @@ public OnBridgeUntilMovement(boolean gridside) {
     robotSpeed = 2 * (gridside ? 1 : -1) * (DriverStation.getAlliance() == Alliance.Red ? -1 : 1);
     backSpeed = 0.4 * (gridside ? -1 : 1) * (DriverStation.getAlliance() == Alliance.Red ? -1 : 1);
 
+    if((gridside == true && DriverStation.getAlliance() == Alliance.Blue) || (gridside == false && DriverStation.getAlliance() == Alliance.Red)){
```

Comment:
You can simplify this:

```
(gridside && DriverStation.getAlliance() == Alliance.Blue) || (!gridside && DriverStation.getAlliance() == Alliance.Red)
```

you don't need to compare booleans to `true` or `false`.